src/workspaces/features/SidebarMenu/SidebarMenu.tsx

```diff
@@ -96,6 +135,7 @@ const SidebarMenu = (props: SidebarMenuProps) => {
         leave="transition ease-in duration-75"
         leaveFrom="transform opacity-100 scale-100"
         leaveTo="transform opacity-0 scale-95"
+        afterLeave={() => showMore(5)}
```

Comment:
What does it do ?

Reply:
It allows to reset the number of displayed workspaces when the user close the sidebar menu.

Reply:
It allows to reset the number of displayed workspaces when the user close the sidebar menu.